.github/workflows/zero-downtime-tests.yml

```diff
@@ -1,13 +1,456 @@
-name: Zero-Downtime Tests
+name: ZeroDowntime Tests
 on:
-  workflow_dispatch:
+  workflow_dispatch: # run integration tests only when triggered manually
+    inputs:
+      branch:
+        description: 'Take CI build artifacts from branch (e.g., master, release-x.y.z)'
```

Comment:
please explain what this input now does in the new pipeline (keptn being upgraded from this version)